### PR TITLE
[157] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by HalimahOladimeji**
Laptop view
Screenshot 2020-12-04 010835
Screenshot 2020-12-04 145716
Tablet view
Screenshot 2020-12-04 150525
Screenshot 2020-12-04 150611
Mobile view
Screenshot 2020-12-04 154837
Screenshot 2020-12-04 154554

### CODE REVIEW

**Review by Ireti203 — COMMENTED** [review]
Your folders should be in the folder with your name inside the submissions folder.

**Comment by HalimahOladimeji** [comment]
I have done that

**Review by Ireti203 — COMMENTED** [review]
Remove the spaces in your file names and image names.

File: submissions/HalimahOladimeji/HTML-forms/contactus.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <p id="copyright"> &copy; Copyright 2020 premium.com <span> All rights reserved.</span></p>
+    </footer>
+</body>
+
+>>>>>>> 8a3e7e9664e3752e7dac92758b7f092c26e53297
+</html>
```
Comment (Ireti203) [inline submissions/HalimahOladimeji/HTML-forms/contactus.html:178]: Did you check this file before pushing?

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Kindly check all your files and ensure you are pushing the right content. Also, you should check your webpage i.e the view after making changes. I am sure your work must have been scattered and will differ from the screenshot you shared

**Review by Ireti203 — COMMENTED** [review]
You ought not to delete the .gitignore file in the submissions folder

**Comment by HalimahOladimeji** [comment]
Corrected

**Review by Ireti203 — APPROVED** [review]

File: submissions/HalimahOladimeji/HTML-forms/product.html [hunk 2]
```diff
@@ -9,8 +9,12 @@
     <div class="topnav">
                     <a class="active" href="#home">Home</a>
                     <a href="#category">Category</a>
+<<<<<<< HEAD
                     <a href="../HTML-forms/product.html">Product</a>
-                    <a href="../HTML-forms/contactformpage.html">Contact Form</a>
+=======
```
Comment (Ireti203) [inline submissions/HalimahOladimeji/HTML-forms/product.html:14]: Remove this

File: submissions/HalimahOladimeji/HTML-forms/product.html [hunk 3]
```diff
@@ -9,8 +9,12 @@
...
+                    <a href="../HTML-forms/index.html">Product</a>
+>>>>>>> 6094bcdff3ddcc5d0b88d4ca3f40cd613674b8bb
```
Comment (Ireti203) [inline submissions/HalimahOladimeji/HTML-forms/product.html:16]: And this also..

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by HalimahOladimeji** [comment]
Done

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**